py3status/modules/net_rate.py

```diff
@@ -59,6 +48,19 @@ def __init__(self, *args, **kwargs):
             value - value (float)
             unit - unit (string)
         """
+        self.all_interfaces = True
+        self.cache_timeout = 2
+        self.devfile = '/proc/net/dev'
+        self.format = "{interface}: {total}"
+        self.format_no_connection = ''
+        self.hide_if_zero = False
+        self.interfaces = ''
+        self.interfaces_blacklist = 'lo'
+        self.precision = 1
+        self.color = False
```

Comment:
Color is sort of a reserved config parameter as it is used to color module output.

I'd be inclined to just make things colored or change the option.  I think this is a topic of debate.  Personally I'm inclined to use the `colors` setting in the general section of the i3status.config to decide if to use colors or not.  If we go for that we can just strip modules color settings anyhow at the low level.